### DISCUSSION TITLE
The Car does not appear

### DISCUSSION THREAD

**Discussion by doganmavuzer** [microsoft/AirSim#4531] [Support Q&A]
Hi, 

When I start the simulation and choose the car, I m facing this issue. There is no Car. 

<img width="1440" alt="Ekran Resmi 2022-05-18 23 52 51" src="https://user-images.githubusercontent.com/22717505/169162141-361ab4e1-e661-44d1-813c-539ef66a9517.png">

{
  "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings.md",
  "SettingsVersion": 1.2,
  "LocalHostIp": "127.0.0.1",
  "RecordUIVisible": true,
  "LogMessagesVisible": true,
  "ViewMode": "",
  "UsageScenario": "",
  "RpcEnabled": true,
  "EnableCollisionPassthrogh": false,
  "PhysicsEngineName": "",
  "Recording": {
    "RecordOnMove": false,
    "RecordInterval": 0.05
  },
  "SceneCaptureSettings" : {
    "Width": 256,
    "Height": 144,
    "FOV_Degrees": 90,
    "AutoExposureSpeed": 100,
    "AutoExposureBias": 0,
    "AutoExposureMaxBrightness": 0.64,
    "AutoExposureMinBrightness": 0.03,
    "MotionBlurAmount": 0,
    "TargetGamma": 1.0
  },  
  "DepthCaptureSettings" : {
    "Width": 256,
    "Height": 144,
    "FOV_Degrees": 90,
    "AutoExposureSpeed": 100,
    "AutoExposureBias": 0,
    "AutoExposureMaxBrightness": 0.64,
    "AutoExposureMinBrightness": 0.03,
    "MotionBlurAmount": 0,
    "TargetGamma": 1.0
  },  
  "SegCaptureSettings" : {
    "Width": 256,
    "Height": 144,
    "FOV_Degrees": 90,
    "AutoExposureSpeed": 100,
    "AutoExposureBias": 0,
    "AutoExposureMaxBrightness": 0.64,
    "AutoExposureMinBrightness": 0.03,
    "MotionBlurAmount": 0,
    "TargetGamma": 1.0
  },
  "Vehicles": {
    "SimpleFlight": {
      "VehicleType": "SimpleFlight",
      "DefaultVehicleState": "Armed",
      "AutoCreate": true,
      "PawnPath": "",
      "EnableCollisionPassthrogh": false,
      "EnableCollisions": true,
      "AllowAPIAlways": true,
      "EnableTrace": false,
      "RC": {
        "RemoteControlID": 0,
        "AllowAPIWhenDisconnected": false
      }
    },
    "PhysXCar": {
      "VehicleType": "PhysXCar",
      "DefaultVehicleState": "",
      "AutoCreate": true,
      "PawnPath": "",
      "EnableCollisionPassthrogh": false,
      "EnableCollisions": true,
      "RC": {
        "RemoteControlID": -1
      }
    }
  },
  "Pixhawk": {
    "LogViewerHostIp": "127.0.0.1",
    "LogViewerPort": 14388,
    "OffboardCompID": 1,
    "OffboardSysID": 134,
    "QgcHostIp": "127.0.0.1",
    "QgcPort": 14550,
    "SerialBaudRate": 115200,
    "SerialPort": "*",
    "SimCompID": 42,
    "SimSysID": 142,
    "SitlIp": "127.0.0.1",
    "SitlPort": 14556,
    "UdpIp": "127.0.0.1",
    "UdpPort": 14560,
    "UseSerial": true,
    "VehicleCompID": 1,
    "VehicleSysID": 135
    }
  }

**Comment by alonfaraj** [comment]
You are missing `SimMode` in settings.json file.
Also, you try to use multirotor and car in the same time which is not possible at the moment.
Take a look in settings documentation:
https://microsoft.github.io/AirSim/settings/

**Reply by doganmavuzer** [reply]
Thank you for your response. I tried this but did not change anything. When I select the multirotor, it works fine but the car does not exist.

**Reply by jonyMarino** [reply]
@doganmavuzer can you share your settings.json including "SimMode": "Car" and taking out multirotor vehicles from it?

**Reply by doganmavuzer** [reply]
Thank you, I fixed the problem after uninstalling and installing again.
